index.html

```diff
@@ -44,202 +42,281 @@ <h1>
-
-              <ul>
-                <li>
-                  <div class="languages-skills">
-                    <img src="./img/Frame7.svg" alt="">  
-                  </div>
-                </li>
-                <li>
-                  <div class="my-skills">
-                    <img src="./img/Frame9(1).svg" alt="" />
-                    <img src="./img/Frame10.svg" alt="" />
-                    <img src="./img/Frame11.svg" alt="" />
-                  </div>
-                </li>
-                <li>
-                  <div class="languages-skills">
-                    <img src="./img/Frame8.svg" alt="">  
-                  </div>
-                </li>
-                <li>
-                  <div class="languages-skills">
-                      <img src="./img/Frame9.svg" alt="">
-                  </div>
-                </li>
-              </ul>
-
-          </section>
+            <ul>
+              <li>
+                <a href="#">
+                  <img src="img/angellist.png" alt="angellist picture" />
+                </a>
+              </li>
+            </ul>
+          </div>
+
+          <button type="button">Get my resume</button>
+
+          <ul>
+            <li>
+              <div class="languages-skills">
+                <img src="./img/Frame7.svg" alt="" />
+              </div>
+            </li>
+            <li>
+              <div class="my-skills">
+                <img src="./img/Frame9(1).svg" alt="" />
+                <img src="./img/Frame10.svg" alt="" />
+                <img src="./img/Frame11.svg" alt="" />
+              </div>
+            </li>
+            <li>
+              <div class="languages-skills">
+                <img src="./img/Frame8.svg" alt="" />
+              </div>
+            </li>
+            <li>
+              <div class="languages-skills">
+                <img src="./img/Frame9.svg" alt="" />
+              </div>
+            </li>
+          </ul>
+        </section>
       </section>
 
+      <section class="contact-me">
+        <h6>Contact me</h6>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it.
+        </p>
+
+        <form action="#" method="post">
+          <input
+            type="text"
+            name="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email-address"
+            placeholder="Email address"
+            required
+          />
+
+          <textarea
+            name="contact"
+            placeholder="Write your message here"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+          ></textarea>
+          <div class="btn">
+            <button type="submit">Get In Touch</button>
+          </div>
+        </form>

```

Comment:
- [ ] Kindly when I tried to submit, the form doesn't get submitted. 
You should use [formspree service](https://formspree.io/html) to submit. 
something like the 
- `action` value should be similar to `https://formspree.io/f/{form_id}`
- `method` value equals `POST`

Reply:
Thank you for the feedback.